src/controllers/user.controller.js

```diff
@@ -11,4 +40,45 @@ export function createUser(req, res){
 
 export function updateUser(req, res){
     
+}
+*/
+
+export const userController = ({ logger, userService }) => {
+    const getUser = async (req, res, next) => {
+        if(!req || (req && !req.auth) || (req && req.auth && !req.auth.id)) {

```

Comment:
This can probably be refactored into
`!req || !req.auth || !req.auth.id`
Due to short circuit evaluation `req.auth.id` even if not set should not cause an error
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#short-circuit_evaluation

Looking at this confirms the two statements are the same 🔥 
![image](https://user-images.githubusercontent.com/17767073/125356434-1abda800-e35e-11eb-92f9-1f56d20584db.png) 🤔